benchmark/inference/inference_benchmark.py

```diff
@@ -145,16 +174,39 @@ def run(args: argparse.ArgumentParser) -> None:
                                 time.reset()
                             with itt, profile:
                                 if args.full_batch:
-                                    full_batch_inference(model, data)
+                                    y = full_batch_inference(model, data)
+                                    if args.evaluate:
+                                        mask = data.test_mask
+                                        pred = y[mask].argmax(1)
+                                        test_acc = pred.eq(data.y[mask]).sum(
+                                        ).item() / mask.sum().item()
+                                        print(f'Full Batch Test Accuracy: \
+                                            {test_acc:.4f}')
                                 else:
-                                    model.inference(subgraph_loader, device,
-                                                    progress_bar=True)
+                                    y = model.inference(
+                                        subgraph_loader, device,
+                                        progress_bar=True)
+                                    if args.evaluate:
+                                        test_acc = model.test(
+                                            y, test_loader, device,
+                                            progress_bar=True)
+                                        print(f'Mini Batch Test Accuracy: \
+                                            {test_acc:.4f}')
 
                         if args.profile:
                             rename_profile_file(model_name, dataset_name,
                                                 str(batch_size), str(layers),
                                                 str(hidden_channels),
                                                 str(num_neighbors))
+                        total_time = time.duration
+                        if args.num_steps != -1:
+                            total_num_samples = args.num_steps * batch_size
+                        else:
+                            total_num_samples = num_nodes
+                        throughput = total_num_samples / total_time
+                        latency = total_time / total_num_samples * 1000
+                        print(f'Throughput: {throughput:.3f} fps')
```

Comment:
```suggestion
                        print(f'Throughput: {throughput:.3f} it/s')
```

Reply:
Here the units should be `samples/s`. My mistake.